dev/io.openliberty.mcp.internal/resources/io/openliberty/mcp/internal/resources/CWMCM.nlsprops

```diff
@@ -155,3 +155,11 @@ jsonrpc.extra.arguments=The following arguments were passed but were not found i
 # Used as part of a response to a client
 # {0} = list of arguments
 jsonrpc.missing.arguments=The following arguments were expected by the method but were not provided: {0}.
+
+# Used for server logging
+# do not translate MCP
+# {0} = The qualified name of the tool
+# {0} = Parameter name
+CWMCM0018E.generic.arguments=CWMCM0018E: The {0} MCP tool method has an argument {1} that contain unsupported components such as TypeVariable, Wildcard, GenericArrayType.
```

Comment:
Suggested change:
The {1} argument of the {0} MCP tool method contains unsupported components such as TypeVariable, Wildcard, GenericArrayType.